src/masonite/stubs/tests/TestCase.py

```diff
@@ -0,0 +1,9 @@
+from masonite.tests import TestCase
```

Comment:
new Masonite projects actually have a base test case inside `tests/TestCase`. this import should be `from tests.TestCase import TestCase`

Reply:
You're right !